diff --git a/src/coq_elpi_vernacular.ml b/src/coq_elpi_vernacular.ml
index ca65350..f136c0a 100644
--- a/src/coq_elpi_vernacular.ml
+++ b/src/coq_elpi_vernacular.ml
@@ -223,8 +223,7 @@ let file_resolver =
       let build_dir = Coq_elpi_config.elpi_dir in
       let installed_dirs =
         let valid_dir d = try Sys.is_directory d with Sys_error _ -> false in
-        let env = Boot.Env.init () in
-        let user_contrib = Boot.Env.(user_contrib env |> Path.to_string) in
+        let user_contrib =  "../.." in
         user_contrib :: Envars.coqpath
         |> List.map (fun p -> p ^ "/elpi/")
         |> ((@) [".";".."]) (* Hem, this sucks *)
diff --git a/theories/elpi.v b/theories/elpi.v
index cf6f7da..387d375 100644
--- a/theories/elpi.v
+++ b/theories/elpi.v
@@ -43,8 +43,3 @@ From Coq Require Bool.
 Register Coq.Bool.Bool.reflect  as elpi.reflect.
 Register Coq.Bool.Bool.ReflectF as elpi.ReflectF.
 Register Coq.Bool.Bool.ReflectT as elpi.ReflectT.
-
-From Coq Require PrimFloat PrimInt63.
-
-Register Coq.Floats.PrimFloat.float as elpi.float64.
-Register Coq.Numbers.Cyclic.Int63.PrimInt63.int as elpi.uint63.
